mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -128,21 +132,10 @@ <h3 class="heading-medium">{% trans 'Credit decisions matching debit card or pri
           </tr>
           <tr class="mtp-check-description-row">
             <td></td>
-            <td colspan="4"><b>Description:</b> {{ credit.security_check.description }}</td>
-          </tr>
-          <tr class="mtp-check-rules-row">
-            <td></td>
-            <td colspan="4"><b>Rules:</b>
-              <ul>
-                {% for rule in credit.security_check.rules %}
-                  <li> {{ rule }} </li>
-                {% endfor %}
-              </ul>
-            </td>
-          </tr>
-          <tr class="mtp-check-decision-row">
-            <td></td>
-            <td colspan="4"><b>Decision Details:</b>
+            <td colspan="4">
+              <strong>Rule:</strong> {{ credit.security_check.description }}
```

Comment:
"Rule" is inconsistent with other views and quite often it's more than one. it should probably be "Rules" or use the number of `credit.security_check.rules` to decide. the list view should be changed to match this too.

also, all text should be marked for translation

Reply:
I think "Rule" is included when the description field is populated in django so don't think the label is needed at all.

Reply:
yes you're right. then this label should be removed